doc/user/.vale.ini

```diff
@@ -0,0 +1,12 @@
+# Vale configuration file.
```

Comment:
I guess the linter will tell you to add the copyright header and a trailing line at the end 😉